src/components/TopView.tsx

```diff
@@ -1,95 +1,315 @@
-  },
-  paper: {
-    flex: 1,
-    minHeight: 0
-  },
-  tabPanel: {
-    flex: 1,
-    display: 'flex',
-    flexDirection: 'column',
-    minHeight: 0,
-    '& > *': {
-      flex: 1,
-      display: 'flex',
-      flexDirection: 'column',
-      minHeight: 0,
-      overflowY: "auto",
-      '& > *': {
+    topView: {
+        display: "flex",
+        flexDirection: "column"
+    },
+    paper: {
         flex: 1,
         minHeight: 0
-      }
+    },
+    tabPanel: {
+        flex: 1,
+        display: "flex",
+        flexDirection: "column",
+        minHeight: 0,
+        "& > *": {
+            flex: 1,
+            display: "flex",
+            flexDirection: "column",
+            minHeight: 0,
+            overflowY: "auto",
+            "& > *": {
+                flex: 1,
+                minHeight: 0
+            }
+        }
     }
-  }
 }));
 
 export function TopView() {
-  const [value, setValue] = useState(0);
-  const handleChange = (event: any, newValue: number) => {
-    setValue(newValue);
-  };
-  const styles = useStyles();
-
-  return (
-    <div className={styles.topView}>
-      <Paper className={styles.topView + ' ' + styles.paper}>
-        <TabContext value={String(value)}>
-          <Tabs textColor="primary" value={value} onChange={handleChange}>
-            <Tab label="Data Visualization" />
-            <Tab label="File Upload" />
-            <Tab label="Tutorials" />
-          </Tabs>
-          <TabPanels className={styles.tabPanel} value={value}>
-            <DataView />
-            <Paper variant="outlined">
-              <FileUpload />
+    const [value, setValue] = useState(0);
+    const [datasets, setDatasets] = useState([]);
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const handleChange = (event: any, newValue: number) => {
+        setValue(newValue);
+    };
+
+    const [user] = useAuthState(firebase.auth());
+
+    const styles = useStyles();
+
+    useEffect(() => {
+        async function getDataSets() {
+            fetch(process.env.NEXT_PUBLIC_GQL_API_UPDATED, {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json",
+                    Authorization: (await getToken()) as unknown as string
+                },
+                body: JSON.stringify({
+                    query: `query GetUser($id: String!) {
+                           user(id: $id) {
+                                 id
+                                 bio
+                                 url
+                                 datasets
+                            }
+                        }
+                    `,
+                    variables: {
+                        id: user.uid
+                    }
+                })
+            })
+                .then((resp) => resp.json())
+                .then((result) => setDatasets(result.data.user.datasets));
+        }
+        globalThis.datasets = datasets;
```

Comment:
If I remember right datasets originally had the actual contents of the datasets. I'm not entirely sure what the graphql query returns but if it only includes the id then this code won't work